.github/workflows/lint.yml

```
@@ -12,8 +12,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: '3.x'
-
+          python-version: ['3.12', '3.13']
```

actionlint (1.7.12)

[error] expected scalar node for string value but found sequence node with "!!seq" tag

(syntax-check)